addons/html_editor/static/src/fields/html_field.js

```diff
@@ -32,14 +46,66 @@ export class HtmlField extends Component {
                 this.lastValue !== record.data[this.props.name].toString()
             ) {
                 this.state.key++;
+                this.state.containsComplexHTML = this.computeContainsComplexHTML();
             }
         });
+
+        useEffect(
+            () => {
+                if (this.iframeRef.el) {
+                    const onLoadIframe = () => this.onLoadIframe();
+                    this.iframeRef.el.addEventListener("load", onLoadIframe);
+                    this.iframeRef.el.after(this.iframeRef.el);
```

Comment:
i dont understand this

Reply:
I forget to had the comment from the old html_field